broker-daemon/orderbook/orderbook-index.js

```diff
@@ -1,39 +1,29 @@
 const { MarketEvent, MarketEventOrder } = require('../models')
-const { migrateStore } = require('../utils')
+const { SubsetStore } = require('../utils')
 
 /**
  * @class Index of active orders based on events received from the Relayer
  */
-class OrderbookIndex {
+class OrderbookIndex extends SubsetStore {
   /**
    * Create a new index from a base store and a store of Market Events
    * @param {sublevel} store      - Base store to sublevel from
    * @param {sublevel} eventStore - Store of MarketEvents to index
    * @param {string} marketName   - Name of the market to index
    */
   constructor (store, eventStore, marketName) {
-    this.store = store.sublevel('orderbook')
-    this.eventStore = eventStore
+    super(store.sublevel('orderbook'), eventStore)
     this.marketName = marketName
   }
 
-  /**
-   * Rebuild the index and add hooks for new events
-   * @returns {void} resolves when the index is rebuilt and ready for new events
-   */
-  async ensureIndex () {
-    await this._clearIndex()
-    await this._rebuildIndex()
-    this._addIndexHook()
-  }
-
   /**
    * Create an object that can be passed to Sublevel to create or remove an orderbook record
+   * @private
    * @param {string} key   - Key of the record to create an index op for
    * @param {string} value - Value of the record being added to the events store to create an index op for
    * @returns {Object} object for create/delete for use with sublevel
    */
-  _addToIndexOperation (key, value) {
+  addToIndexOperation (key, value) {
```

Comment:
Did this API change, or we were just using the internal `_addToIndex..` method instead of the public one?

Reply:
Just changing the style of private methods to be underscore-less.